### PR TITLE
NAS-102847 add docs on helptext in truenas UI

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/snippets/usingui.rst [hunk 1]
```diff
@@ -72,3 +72,15 @@ DNS hostname, or mDNS name:
...
+
+.. _Help Text:
+
+Help Text
+~~~~~~~~~
+
+Every field and setting in the |web-ui| has a |help-text| icon.
+Additional information about the field or setting can be shown by
+clicking |help-text|. The help text is pinned to the screen until
+|help-text| is clicked again. The help text box can be dragged to any
```
Comment (a-stjohn) [inline userguide/snippets/usingui.rst]:
```suggestion
|help-text| is clicked again. The help text dialog can be dragged to any
```

File: userguide/snippets/usingui.rst [hunk 2]
```diff
@@ -72,3 +72,15 @@ DNS hostname, or mDNS name:
...
   :samp:`ping {192.168.1.19}` on the command line of the other
   computer. If there is no response, check network configuration.
+
+
+.. _Help Text:
+
+Help Text
+~~~~~~~~~
+
+Every field and setting in the |web-ui| has a |help-text| icon.
```
Comment (wblock) [inline userguide/snippets/usingui.rst]: We should probably not promise for every field, even though we try to do that.
```suggestion
Most fields and settings in the |web-ui| have a |help-text| icon.
```

File: userguide/snippets/usingui.rst [hunk 3]
```diff
@@ -72,3 +72,15 @@ DNS hostname, or mDNS name:
...
+Additional information about the field or setting can be shown by
+clicking |help-text|. The help text is pinned to the screen until
```
Comment (wblock) [inline userguide/snippets/usingui.rst]: Not clear what "pinned" means here.  Maybe have the sentence about moving first:
The help text window can be dragged to any location, and will remain there until X or ? is clicked to close the window. Multiple help text windows can be open at one time.

File: userguide/Makefile [hunk 4]
```diff
@@ -133,7 +133,7 @@ latexpdf pdf:	requirepdf preproc
 	#|help-pin|
 	@sed -i '' -e 's%%{\\material\\symbol{"F703}}%g' $(PREPROCDIR)/$(BUILDDIR)/latex/*.tex
 	#|help-unpin|
```
Comment (wblock) [inline userguide/Makefile]: This should be renamed to |help-close|.
```suggestion
	#|help-close|
```

File: userguide/conf.py [hunk 5]
```diff
@@ -89,7 +89,7 @@
 .. |dockerhost|           replace:: Docker VM
 .. |help-pin|             replace::  (Pin)
 .. |help-text|            replace::  (Help Text)
-.. |help-unpin|           replace::  (Unpin)
+.. |help-unpin|           replace::  (Close)
```
Comment (wblock) [inline userguide/conf.py]:
```suggestion
.. |help-close|           replace::  (Close)
```

File: userguide/snippets/usingui.rst [hunk 6]
```diff
@@ -125,3 +125,17 @@ DNS hostname, or mDNS name:
...
+
+.. _Help Text:
+
+Help Text
+~~~~~~~~~
+
+Most fields and settings in the |web-ui| have a |help-text| icon.
+Additional information about the field or setting can be shown by
+clicking |help-text|. The help text window can be dragged to any
+location, and will remain there until |help-unpin| or |help-text| is
```
Comment (wblock) [inline userguide/snippets/usingui.rst]:
```suggestion
location, and will remain there until |help-close| or |help-text| is
```

File: userguide/snippets/typography.rst [hunk 7]
```diff
@@ -66,5 +66,5 @@
    +---------------------------+----------------------------------------------------+
    | |help-pin|                | Pin a help box to the screen.                      |
    +---------------------------+----------------------------------------------------+
-   | |help-unpin|              | Unpin a help box from the screen.                  |
+   | |help-close|              | Unpin a help box from the screen.                  |
```
Comment (wblock) [inline userguide/snippets/typography.rst]: White space must be fixed.
```suggestion
   | |help-close|              | Close a help box.                  |
```

File: userguide/snippets/usingui.rst [hunk 8]
```diff
@@ -125,3 +125,17 @@ DNS hostname, or mDNS name:
...
+.. _Help Text:
+
+Help Text
+~~~~~~~~~
+
+Most fields and settings in the |web-ui| have a |help-text| icon.
+Additional information about the field or setting can be shown by
+clicking |help-text|. The help text window can be dragged to any
+location, and will remain there until |help-close| or |help-text| is
+clicked to close the window. The help text is pinned to the screen
```
Comment (wblock) [inline userguide/snippets/usingui.rst]: This sentence about being "pinned" does not make sense and can be removed.

File: userguide/snippets/usingui.rst [hunk 9]
```diff
@@ -125,3 +125,17 @@ DNS hostname, or mDNS name:
...
+until |help-text| is clicked again. The help text dialog can be
+dragged to any position on the page.
```
Comment (wblock) [inline userguide/snippets/usingui.rst]: Already mentioned draggability above, just remove this last sentence.

**Pull request merged**